src/Type/Input/DateQueryInput.php

```diff
@@ -38,12 +38,12 @@ public static function register_type() {
 						'description' => __( 'Second (0 to 59)', 'wp-graphql' ),
 					],
 					'after'     => [
-						'type'        => 'DateInput',
-						'description' => __( 'Nodes should be returned after this date', 'wp-graphql' ),
+						'type'        => 'String',
```

Comment:
I think we need tighter validation here. I think it would be better to introduce a new DateTime scalar: https://webonyx.github.io/graphql-php/type-system/scalar-types/#writing-custom-scalar-types and use that here instead.